.travis.yml

```diff
@@ -11,7 +11,11 @@ matrix:
         - |
           bash .tools/check_style.sh
           RESULT=$?; if [ $RESULT -eq 0 ]; then true; else false; fi;
-        - cd go && bash .tools/gen_config.sh && glide install --strip-vendor && go test $(glide novendor)
+        - ln -s $GOPATH/src/github.com/PaddlePaddle $GOPATH/src/github.com/paddlepaddle
+        - cd go && glide install && go get k8s.io/kubernetes || echo 1
+        - bash ./vendor/k8s.io/code-generator/generate-groups.sh "deepcopy,client,informer,lister" github.com/PaddlePaddle/cloud/go/pkg/client github.com/PaddlePaddle/cloud/go/pkg/apis paddlepaddle:v1alpha1
+        - grep "github.com/paddlepaddle/cloud" -nR pkg/client | awk -F ':' '{print $1}' | xargs sed -i 's|github.com/paddlepaddle/cloud|github.com/PaddlePaddle/cloud|g'
+        - bash .tools/gen_config.sh && glide install --strip-vendor && go test $(glide novendor)
```

Comment:
chang `github.com/paddlepaddle/cloud` to `github.com/PaddlePaddle/cloud` to avoid `case-insensitive import collision`